src/state/trajectory/logics.ts

```diff
@@ -575,6 +578,35 @@ const convertFileLogic = createLogic({
     type: CONVERT_FILE,
 });
 
+const receiveConvertedFileLogic = createLogic({
+    process(deps: ReduxLogicDeps, dispatch, done) {
+        const { action, getState } = deps;
+        const currentState = getState();
+        const conversionStatus = getConversionStatus(currentState);
+        const simulariumController = getSimulariumController(currentState);
+        const simulariumFile = action.payload;
+
+        simulariumController.changeFile(
+            netConnectionSettings,
+            simulariumFile.name,
+            true
+        );
+
+        clearOutFileTrajectoryUrlParam();
+        history.replaceState(
+            {},
+            "",
+            `${location.origin}${location.pathname}?${URL_PARAM_KEY_FILE_NAME}=${simulariumFile.name}`
+        );
+        if (conversionStatus !== CONVERSION_INACTIVE) {
+            dispatch(setConversionStatus({ status: CONVERSION_INACTIVE }));
+        }
+        simulariumController.gotoTime(0);
+        done();
```

Comment:
goToTime is no longer called during the controller convertTrajectory method (formerly convertAndLoadTrajectory)